tests/python/test_legacy.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+import pytest
+from util import Mock, roll
+
+@pytest.mark.parametrize("r,out,mock,mock_const",[
+    ("2d2d2d==1", 3, Mock.RETURN_INCREMENTING, 1),
+])
+def test_rerolling_multidice(r, out, mock, mock_const):
+    result = roll(r, mock_mode=mock, mock_const=mock_const)
+    assert result == out
```

Comment:
The test fails (as expected), so we should check that it raises an Exception.
See: https://docs.pytest.org/en/6.2.x/assert.html
> with pytest.raises(Exception):

There should be other examples of expected failures in the tests